static/js/search_suggestion.js

```diff
@@ -40,6 +40,37 @@ function match_criteria(operators, criteria) {
     });
 }
 
+function compare_by_huddle(huddle) {
+    huddle = _.map(huddle.slice(0, -1), function (person) {
+        person = people.get_by_email(person);
+        if (person) {
+            return person.user_id;
+        }
+    });
+
+    // Construct dict for all huddles, so we can lookup each's recency
+    var huddles = activity.get_huddles().slice(0, 10);
```

Comment:
why the slice here?  It seems like we want to slice at the end (after doing the filtering for whether they match), not here at the beginning.